tests/unit/objects/test_project_import_export.py

```diff
@@ -53,6 +53,30 @@ def resp_remote_import():
         yield rsps
 
 
+@pytest.fixture
+def resp_remote_import_s3():
+    content = {
+        "id": 1,
+        "description": None,
+        "name": "remote-project-s3",
+        "name_with_namespace": "Administrator / remote-project-s3",
+        "path": "remote-project-s3",
+        "path_with_namespace": "root/remote-project-s3",
+        "created_at": "2018-02-13T09:05:58.023Z",
+        "import_status": "scheduled",
+    }
```

Comment:
I just realized that, at least in the upstream examples, all import status responses are the same, so we could just reuse a module-level `import_content` in all the fixtures IMO. But not a blocker here.

Reply:
Yeah have observed that as well. But I don't really try to refactor test code xD. Will try

Reply:
Not done ;_;. Maybe we can tackle this in a separate  refactor ticket. I don't want to lose uniformity of each test having its own response

Reply:
Makes sense :)